specs/passenger_spec.rb

```diff
@@ -75,4 +75,40 @@
       end
     end
   end
+
+  describe "total_spent method" do
+    it "calculates total spent" do
+      trip_dispatcher = RideShare::TripDispatcher.new
+      expected_result = 37.95
+
+      passenger = trip_dispatcher.find_passenger(35)
+
+      passenger.total_spent.must_equal expected_result
+
+    end
+
+    it "does not calculate total of an in-progress trip" do
+      trip_dispatcher = RideShare::TripDispatcher.new
+      expected_result = 37.95
+
```

Comment:
Good work catching this somewhat tricky edge case. It seems to me like this would apply to all the methods from wave 1.